tests/get_executable_path_test.c

```diff
@@ -70,9 +70,12 @@ CTEST(tests_exe_path_return, invalid_command)
 	ASSERT_NULL(get_executable_path("      "));
 }
 
-CTEST(tests_exe_path_return, path_for_valid_commands)
-{
-	char *path = get_executable_path("ls");
-	ASSERT_NOT_NULL(path);
-	free(path);
-}
+/*
+	commented this test because we are now using our own env
+*/
+// CTEST(tests_exe_path_return, path_for_valid_commands)
```

Comment:
So we can remove it for good, can you do it please?

Reply:
I'm scared of just removing tests ;)